PWGJE/EMCALJetTasks/macros/AddTaskRhoSparseSV.C

```diff
@@ -0,0 +1,95 @@
+// $Id$
+
+AliAnalysisTaskRhoSparseSV* AddTaskRhoSparseSV(
+					   const char    *nJetsBkg    = "JetsBkg",
+					   const char    *nJetsSig    = "JetsSig",
+					   const char    *nTracks     = "PicoTracks",
+					   const char    *nClusters   = "CaloClusters",  
+					   const char    *nRho        = "Rho",
+					   Double_t       jetradius   = 0.2,
+					   const char    *cutType     = "TPC",
+					   Double_t       jetareacut  = 0.01,
+					   Double_t       jetptcut    = 0.0,
+					   Double_t       emcareacut  = 0,
+					   TF1           *sfunc       = 0x0,
+					   const UInt_t   exclJets    = 2,
+					   const Bool_t   histo       = kFALSE,
+					   const char    *taskname    = "Rho",
+					   const Bool_t   fRhoCMS      = kTRUE
+					   )
+
+{  

```

Comment:
Better use static add function within the class as you find it in most classes of the PWGJE library: This simplifies ROOT6 compatibility and forces the code to be compiled which makes it more robust. In addition code in libraries allow full usage of the c++ standard and not only what is supported by CINT.